### PR TITLE
ENH: Added histo ssl configs

### PR DESCRIPTION
**Pull request by maxilse**
Deleted covid and cifar related files. Added ssl histo configs.

### CODE REVIEW

File: hi-ml-histopathology/src/SSL/configs/HistoSimCLRContainer.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+from typing import Any, Optional, Tuple, List
```
Comment (harshita-s) [inline hi-ml-histopathology/src/SSL/configs/HistoSimCLRContainer.py:1]: I think the copyright statement needs to be added here. same for other files.

File: hi-ml-histopathology/src/SSL/configs/PANDA_SimCLRContainer.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+from enum import Enum
```
Comment (harshita-s) [inline hi-ml-histopathology/src/SSL/configs/PANDA_SimCLRContainer.py:1]: Please add copyright statement and docstrings (if required)

File: hi-ml-histopathology/src/SSL/configs/PANDA_SimCLRContainer.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+if local_mode:
+    is_debug_model = True
+    num_workers = 0
+
+else:
+    is_debug_model = False
+    num_workers = 5
+
+
+class SSLDatasetNameRadiomicsNN(SSLDatasetName, Enum):
```
Comment (harshita-s) [inline hi-ml-histopathology/src/SSL/configs/PANDA_SimCLRContainer.py:26]: why this class name?

File: hi-ml-histopathology/src/SSL/configs/CRCK_SimCLRContainer.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+local_mode = False
+path_local_data: Optional[Path]
+if local_mode:
+    is_debug_model = True
+    num_workers = 0
+    max_epochs = 2
+else:
+    is_debug_model = False
+    num_workers = 12
+    max_epochs = 200
```
Comment (harshita-s) [inline hi-ml-histopathology/src/SSL/configs/CRCK_SimCLRContainer.py:24]: I think this block (line 15-24) can be changed in the same way as we did for the deepmil configs. Happy to discuss if there are questions

File: hi-ml-histopathology/src/SSL/configs/CRCK_SimCLRContainer.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+
+class SSLDatasetNameRadiomicsNN(SSLDatasetName, Enum):
```
Comment (harshita-s) [inline hi-ml-histopathology/src/SSL/configs/CRCK_SimCLRContainer.py:27]: not sure why we have this class name

File: hi-ml-histopathology/src/histopathology/configs/run_ids.py [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # TODO: remove innereye references
 innereye_ssl_checkpoint = "hsharma_panda_explore:hsharma_panda_explore_1638437076_357167ae"
 innereye_ssl_checkpoint_binary = "hsharma_panda_tiles_ssl:hsharma_panda_tiles_ssl_1639766433_161e03b9"
-innereye_ssl_checkpoint_crck_4ws = "ModifyOldSSLCheckpoint:a9259fdb-3964-4c5b-8962-4660e0b79d44"
+innereye_ssl_checkpoint_crck_4ws = "simclr_8_gpus:CRCK_SimCLR_1647951559_d2a19df5"
 innereye_ssl_checkpoint_crck_radiomics = "ModifyOldSSLCheckpoint:704b1af8-7c75-46ed-8460-d80a0e603194"
```
Comment (harshita-s) [inline hi-ml-histopathology/src/histopathology/configs/run_ids.py:5]: do we still need the last checkpoint?

File: hi-ml-histopathology/src/SSL/configs/CRCK_SimCLRContainer.py [hunk 7]
```diff
@@ -0,0 +1,60 @@
+from enum import Enum
```
Comment (harshita-s) [inline hi-ml-histopathology/src/SSL/configs/CRCK_SimCLRContainer.py:1]: Could we check the config for few epochs to see if it runs without error?

File: hi-ml-histopathology/src/SSL/configs/PANDA_SimCLRContainer.py [hunk 8]
```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+from pathlib import Path
+from typing import Any, Optional
```
Comment (harshita-s) [inline hi-ml-histopathology/src/SSL/configs/PANDA_SimCLRContainer.py:3]: Could we check the PANDA config for few epochs to see if it runs without error?

File: hi-ml-histopathology/src/histopathology/configs/run_ids.py (hunk 6 above)
Comment (maxilse) [reply hi-ml-histopathology/src/histopathology/configs/run_ids.py:5]: yes

**Review by harshita-s — APPROVED** [review]

**Review by mebristo — APPROVED** [review]

**Pull request merged**